docs/imviz/plugins.rst

```diff
@@ -196,6 +196,24 @@ results are displayed under the :guilabel:`CALCULATE` button.
     :ref:`Export Photometry <imviz_export_photomtery>`
         Documentation on exporting photometry results.
 
+.. _imviz-catalogs:
+
+Catalog Search
+==============
+
+This plugin queries a specified region of the image, marks the sources from the results of the query, and
+provides the number of sources found. After zooming into a specific region of the image, the query uses the center
```

Comment:
this makes it sound a little like it might take a subset as input, but as far as I understand, it just uses the current viewer zoom-limits, right?  Maybe something like the following would avoid that misinterpretation:

```suggestion
This plugin queries a catalog around the coordinates of the active image, marks the sources from the results of the query, and
provides the number of sources found. After zooming into a specific region of the image, the query uses the center
```

Reply:
Correct, it's using the current viewer's zoom-limits! I can see where it can sound like it requires a subset, so I'll make those changes, thank you!